### PR TITLE
Sdk bump v0.0.7 => v0.7.0

### PR DESCRIPTION
**Pull request by ewolinetz**
Depends on https://github.com/openshift/elasticsearch-operator/pull/122

### CODE REVIEW

**Comment by ewolinetz** [comment]
currently troubleshooting adding the elasticsearch scheme for the operator to run

**Comment by ewolinetz** [comment]
looks like fluentd isn't correctly being deployed/tested as part of the ci and this is hidden due to running tests in parallel. tests will begin to start failing for `e2e-operator` please do not retest

**Comment by ewolinetz** [comment]
@jcantrill I'm currently looking into how we can do mocking with our tests and can make that part of this PR.

File: pkg/controller/clusterlogging/clusterlogging_controller.go [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+var log = logf.Log.WithName("controller_clusterlogging")
+
+const (
+	singletonName    = "instance"
+	singletonMessage = "ClusterLogging is a singleton. Only an instance named 'instance' is allowed"
+)
+
+/**
+* USER ACTION REQUIRED: This is a scaffold file intended for the user to modify with their own Controller
```
Comment (josefkarasek) [inline pkg/controller/clusterlogging/clusterlogging_controller.go]: I think we can delete this

File: pkg/controller/clusterlogging/clusterlogging_controller.go [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+	// Watch for changes to primary resource ClusterLogging
+	err = c.Watch(&source.Kind{Type: &loggingv1.ClusterLogging{}}, &handler.EnqueueRequestForObject{})
+	if err != nil {
+		return err
+	}
+
+	// TODO(user): Modify this to be the types you create that are owned by the primary resource
+	// Watch for changes to secondary resource Pods and requeue the owner ClusterLogging
+	err = c.Watch(&source.Kind{Type: &corev1.Pod{}}, &handler.EnqueueRequestForOwner{
```
Comment (josefkarasek) [inline pkg/controller/clusterlogging/clusterlogging_controller.go]: Why do we watch for pod events? What actions are supported here?

File: pkg/k8shandler/status.go [hunk 3]
```diff
@@ -197,9 +208,9 @@ func getReplicaSetNames(node elasticsearch.ElasticsearchStatus) []string {
 
 	replicasetNames := []string{}
 
-	for _, nodeStatus := range node.Nodes {
+	/*for _, nodeStatus := range node.Nodes {
```
Comment (josefkarasek) [inline pkg/k8shandler/status.go]: Should this function be implemented or removed?

**Comment by josefkarasek** [comment]
Outstanding job, Eric!! Really good!

I only stumbled upon some unexpected comments. Please run `go fmt` and we can merge.

File: pkg/k8shandler/status.go (hunk 3 above)
Comment (ewolinetz) [reply pkg/k8shandler/status.go]: i believe removed, not sure if this is cruft from the status update

**Comment by ewolinetz** [comment]
I believe I need to update this PR to point to `master` instead of `master-post-release`

**Pull request merged**